features/unit/v2indexerclient_paths.feature

```diff
@@ -218,7 +218,7 @@ Feature: Indexer Client v2 Paths
       | /v2/applications/1234/logs                                                                                                                                                                                    | 1234           | 0     | 0        | 0        |           |                                                            |                                                      |
       | /v2/applications/1234/logs?limit=4&max-round=120&min-round=100&next=TOKEN&sender-address=PNWOET7LLOWMBMLE4KOCELCX6X3D3Q4H2Q4QJASYIEOF7YIPPQBG3YQ5YI&txid=TDIO6RJWJIVDDJZELMSX5CPJW7MUNM3QR4YAHYAKHF3W2CFRTI7A | 1234           | 4     | 100      | 120      | TOKEN     | PNWOET7LLOWMBMLE4KOCELCX6X3D3Q4H2Q4QJASYIEOF7YIPPQBG3YQ5YI | TDIO6RJWJIVDDJZELMSX5CPJW7MUNM3QR4YAHYAKHF3W2CFRTI7A |
 
-  @unit.indexer.ledger_refactoring
```

Comment:
are these tags being used in the sdks? if so, will removing these break the tests?

Reply:
Yes, this will break the tests and this necessitates removing the same tags from the SDK's.

Reply:
Does it break them, or prevent them from running?

There are community SDKs that implement these.

Reply:
I didn't realize there are community SDK's that also implement cuke tests. because of that I'll revert.

And you're right, that four our SDK's nothing would break since the tests will still run under a different tag and we know that the necessary steps have already been implemented.